<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PdfViewer</class>
 <widget class="QWidget" name="PdfViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PdfViewer</string>
  </property>
  <property name="windowIcon">
   <iconset theme=":/resources/images/pdfLogo.png">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="m_previousPageBtn">
       <property name="text">
        <string>...</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_nextPageBtn">
       <property name="text">
        <string>...</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_shareBtn">
       <property name="text">
        <string>...</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_cropBtn">
       <property name="text">
        <string>...</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_extractTextBtn">
       <property name="text">
        <string>...</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_exportToMapBtn">
       <property name="text">
        <string>...</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_exportToImageBtn">
       <property name="text">
        <string>...</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_continuousBtn">
       <property name="text">
        <string>...</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="m_zoomInBtn">
       <property name="text">
        <string>...</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_zoomLevel">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>Fit width</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fit in View</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>25%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>50%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>75%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>100%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>150%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>200%</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_zoomOutBtn">
       <property name="text">
        <string>...</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSpinBox" name="m_curentPage">
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_pageCount">
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="m_splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="bookmarkTab">
       <attribute name="title">
        <string>Bookmarks</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QTreeView" name="bookmarkView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="headerHidden">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pagesTab">
       <attribute name="title">
        <string>Pages</string>
       </attribute>
      </widget>
     </widget>
     <widget class="QPdfView" name="m_view" native="true"/>
    </widget>
   </item>
  </layout>
  <action name="m_cropViewAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="phone">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Crop View</string>
   </property>
  </action>
  <action name="m_shareAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="document-share">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Share</string>
   </property>
  </action>
  <action name="m_exportToMapAct">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="m_exportToImage">
   <property name="text">
    <string>Image</string>
   </property>
  </action>
  <action name="m_continuousAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Continuous</string>
   </property>
  </action>
  <action name="m_zoomInAct">
   <property name="icon">
    <iconset theme="zoom-in">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="m_zoomOutAct">
   <property name="icon">
    <iconset theme="zoom-out">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="m_previousPageAct">
   <property name="icon">
    <iconset theme="media-skip-backward">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Previous Page</string>
   </property>
  </action>
  <action name="m_nextPageAct">
   <property name="icon">
    <iconset theme="media-skip-forward">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Next Page</string>
   </property>
  </action>
  <action name="m_extractTextAct">
   <property name="text">
    <string>Text</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QPdfView</class>
   <extends>QWidget</extends>
   <header location="global">qpdfview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
